.github/workflows/tests.yaml

```diff
@@ -1,8 +1,11 @@
 name: Integration testing
 
 on:
-  workflow_dispatch:
   pull_request:
+  push:
+    branches:
+      - main
```

Comment:
Does this mean the push event is restricted to the "main" branch? Do we intend to create a "main" branch?

Reply:
@antonbaliasnikov there will be no main branch, only the base.

Reply:
Good catch! Removed this.